### ISSUE TITLE
respimg doing even vertical resizing?

### ISSUE THREAD

**Issue by stratboy** [aFarkas/lazysizes#638]
Hi, I was using a simple code like this:

``` html
          <img
            class="lazyload"
            data-src="<?= $image_medium; ?>"
            data-srcset="<?= $image_medium; ?> 960w, <?= $image_large; ?> 1400w"
            data-sizes="auto"
          />
```

But that caused the image to also resize based on window height (very strange), making that code unusable.

To have a rebuttal, I used plain html code, and this works perfect, but then I can't use lazysizes:

``` html
          <img
            src="<?php echo $image_medium; ?>"
            srcset="<?php echo $image_medium ?> 960w, <?php echo $image_large ?> 1400w"
            sizes="(max-width:960px) 960px, (max-width:4500px) 1400px, 1400px"
          />
```

So what am I doing wrong or what should I do?

**Comment by aFarkas** [comment]
What do you mean with "caused the image to also resize based on window height"? Does this happen in every browser. Can you create a testcase or explain more in depth.

In general lazySizes re-computes the `sizes` attribute on every `window.onresize` event. If the image doesn't change it. The browser should select the same image otherwise it has to load another one. For me this looks ok.

**Comment by stratboy** [comment]
Hi, I mean that strangely enough, the image was downsized even in the height side when resizing the window only by height. Normal native behavior is that image will resize only if the window is resized even by width.

Anyway, I solved my issue using piturefill, so I can make full use of the sizes conditions (Since I think the issue was related to how respimg auto-set the sizes property > a this in turn was probably related to how I wrote my css, since I can't really predict the size ratio).
